platform-manageability-agent/cmd/platform-manageability-agent/platform-manageability-agent.go

```diff
@@ -160,6 +262,34 @@ func main() {
 	log.Infof("Platform Manageability Agent finished")
 }
 
+func enableService(action, service string) error {
+	allowedActions := map[string]bool{"enable": true, "start": true}
+	allowedServices := map[string]bool{"lms.service": true}
+
+	if !allowedActions[action] {
+		return fmt.Errorf("invalid action: %v", action)
+	}
+	if !allowedServices[service] {
+		return fmt.Errorf("invalid service: %v", service)
+	}
+
+	cmd := exec.Command("sudo", "systemctl", action, service)
```

Comment:
Instead of using `exec.Command` directly, this should use the function provided by the utils function